webdriver/tests/bidi/browsing_context/capture_screenshot/clip.py

```diff
@@ -1,4 +1,6 @@
 import pytest
+
+import webdriver.bidi.error as error
 from tests.support.image import png_dimensions
 from webdriver.bidi.modules.browsing_context import ElementOptions, BoxOptions
 from webdriver.bidi.modules.script import ContextTarget
```

Comment:
`webdriver` is still a third-party package while tests can be considered local imports. Maybe you can fix that as ride-along?

```suggestion
import webdriver.bidi.error as error
from webdriver.bidi.modules.browsing_context import ElementOptions, BoxOptions
from webdriver.bidi.modules.script import ContextTarget

from tests.support.image import png_dimensions

```